README.md

```diff
@@ -135,7 +135,8 @@ Plugins which allow importing other types of files as modules.
 - [glsl](https://github.com/vwochnik/rollup-plugin-glsl) - Import GLSL shaders as compressed strings.
 - [glsl-optimize](https://github.com/docd27/rollup-plugin-glsl-optimize) - Import and optimize GLSL shaders as compressed strings. Supports [glslify](https://github.com/glslify/glslify).
 - [glslify](https://github.com/glslify/rollup-plugin-glslify) - Import GLSL strings with [glslify](https://github.com/glslify/glslify).
-- [gltf](https://github.com/bengsfort/rollup-plugin-gltf) - Import glTF 3d models as a URI or inlined JSON.
+- [gltf](https://github.com/nytimes/rd-bundler-3d-plugins) - Import, transform, optimize, and compress glTF 3D models.
+- [gltf-inline](https://github.com/bengsfort/rollup-plugin-gltf) - Import glTF 3D models as a URI or inlined JSON.
```

Comment:
```suggestion
```

We don't know that this will be renamed. If the NPM package name has been handed off, this will not be useful to users. If I were the owner of that package, I would have insisted that the functionality be _extended_ rather than _overwritten_, alas.

Reply:
> I would have insisted that the functionality be extended rather than overwritten, alas.

I feel that we made a reasonable decision here in terms of functionality and naming for the new plugin, but point taken — if you'd prefer to remove the older repository from the list unless/until it is re-published on NPM, we can do that.